WasatchNET/EEPROM.cs

```diff
@@ -881,6 +881,39 @@ public byte startupScansToAverage
         }
         byte _startupScansToAverage;
 
+        public byte matchingMinRampPixels
+        {
+            get { return _matchingMinRampPixels; }            
+            set
+            {
+                _matchingMinRampPixels = value;
+                EEPROMChanged?.Invoke(this, new EventArgs());
+            }
+        }
+        byte _matchingMinRampPixels = 10;

```

Comment:
I'm curious at what this is meant to do. I should probably look at the previous PR...

Reply:
Technically it's just a named storage field, that doesn't "do" anything in the driver.  However, it does do something in the spectrometer (at least some models :-)  Can discuss more offline.